Resources/Prototypes/Entities/Structures/Machines/research.yml

```
@@ -57,6 +57,9 @@
   - type: GuideHelp
     guides:
     - Science
+  - type: ResearchPointSource # Goobstation
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)